public/data/kumarsonsoff3.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "Priyanshu Kumawat",
+  "type": "personal",
+  "bio": "Web developer | Tech writer | Open Source Enthusiast!",
+  "avatar": "https://github.com/kumarsonsoff3.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/kumarsonsoff3",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Kumar_Sons_off",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Completed JavaScript Udemy course",
+      "date": "December 2021",
+      "icon": "globe",
+      "color": "green",
+      "description": "Completed JavaScript Udemy course by Jonas schmedtmann"
+    },
+    {
+      "title": "Started Freelancing",
+      "date": "November 2021",
+      "icon": "dollar",
+      "color": "gold",
+      "description": "Started freelancing"
+    },
+    {
+      "title": "Published Firt blog on Hashnode",
```

Minor spelling error here. Might want to create a good first issue for this. 🙂
```suggestion
      "title": "Published my first blog on Hashnode",
```